src/main/kotlin/no/nav/syfo/minesykmeldte/model/PreviewSykmeldt.kt

```diff
@@ -17,6 +17,7 @@ data class PreviewSykmeldt(
     val dialogmoter: List<Dialogmote>,
     val aktivitetsvarsler: List<Aktivitetsvarsel>,
     val oppfolgingsplaner: List<Oppfolgingsplan>,
+    val isPilotUser: Boolean,
```

Comment:
Er det mulig å rename det slik at det blir tydelig hva man er pilot for? Antar kanskje det er oppfølgingsplan?

Reply:
Done